content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md

```diff
@@ -79,6 +82,18 @@ Where applicable, it links to outstanding issues on GitHub.
 * [Does the precision of the timestamp matter?](#does-the-precision-of-the-timestamp-matter)
 * [What are the configuration recommendations and schema guidelines for writing sparse, historical data?](#what-are-the-configuration-recommendations-and-schema-guidelines-for-writing-sparse-historical-data)
```

Comment:
@pierwill, fyi, pushed up a link fix for "Why aren't data dropped after I've altered..."

Reply:
@pierwill, looks like there's a handful more link fixes we need to remove hyphens for...let's do a global search and fix, and then re-run a link audit at last stop before we push Enterprise 1.9 PR